types/react/index.d.ts

```diff
@@ -3749,6 +3751,153 @@ declare namespace React {
         zoomAndPan?: string | undefined;
     }
 
+    // https://www.w3.org/TR/mathml-core/#global-attributes
+    interface MathMLAttributes<T> extends DOMAttributes<T> {
+        className?: string | undefined;
+        dir?: "ltr" | "rtl" | undefined;
+        displaystyle?: boolean | undefined;
```

Comment:
This currently warns for booleans as far as I can tell: https://codesandbox.io/p/sandbox/tender-feistel-ym2mcg?file=%2Fsrc%2Findex.js%3A11%2C23. I think you need to use `"false" | "true"` for now.

Reply:
Nice catch.

* Types updated in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/71187/commits/ae57a4d5babf168190484685d50d081b77f3647b
* Tests related to this updated in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/71187/commits/c8b502aba0df8d12888a2da49d7bc1a6722b08d8